quartical/apps/backup.py

```diff
@@ -86,7 +94,8 @@ def restore():
     restored_xds_list = xds_to_storage_table(
         zarr_xds_list,
         args.ms_path,
-        columns=(args.column,)
+        columns=(args.column,),
+        rechunk=True
```

Comment:
Rechunking here _should_ be unnecessary as we are always writing to a new location (base of datetime).

Reply:
I'm not sure I follow. We are possibly writing to a dataset with different chunks on disk so I think rechunking *might* be necessary

Reply:
Ah sorry, I misread this - thought it was in backup. Yeah - absolutely. Going back it is a must.